src/parse/mod.rs

```diff
@@ -488,4 +490,39 @@ pub(crate) mod tests {
         let buf: String = buf.into_iter().collect();
         assert!(parse(&buf).is_none());
     }
+
+    prop_compose! {
+        fn arb_vec_result_locatable_token()(tokens in any::<Vec<Token>>()) -> Vec<CompileResult<Locatable<Token>>> {
+            tokens.into_iter().map(|token| Ok(Locatable { data: token, location: Location::default()})).collect()
+        }
```

Comment:
I don't think this should be its own `prop!` implementation, it never generates any error variants. Could you make it a free function instead?

Reply:
Actually I changed my mind - we can just add error variants in the future.